lib/eth-watcher.hoon

```diff
@@ -0,0 +1,38 @@
+::  eth-watcher utilities
```

Comment:
Why are these two in a separate lib?  Do we expect clients to use them?

Reply:
I imagine clients may want `++store-new-logs`, yes. `++log-to-id` is just there to support it, though maybe it belongs in zuse instead?